### DISCUSSION TITLE
app_url config not working after upgrade

### DISCUSSION THREAD

**Discussion by shlucky90** [livewire/livewire#5471] [7. Bugs (archived)]
After upgrading to 2.10.8, I am getting 404 error everywhere which was perfectly working fine with `app_url` custom link `https://myurl.com/crm/index.php`

**Comment by lam0819** [comment]
Could you check, is it related to  https://github.com/livewire/livewire/pull/4570

**Comment by joshhanley** [comment]
@shlucky90 there was a bug released in v2.10.8. I have submitted a PR #5490 with a fix. Hope this helps!
